src/Integration.UnitTests/Persistence/SolutionBindingSerializerTests.cs

```diff
@@ -356,7 +354,7 @@ public void SolutionBindingSerializer_WriteSolutionBinding_AddConfigFileToSoluti
 
         private SolutionBindingSerializer CreateTestSubject()
         {
-            return new SolutionBindingSerializer(this.serviceProvider, this.store, new SonarLintOutputLogger(serviceProvider));
+            return new SolutionBindingSerializer(this.serviceProvider, this.store, new SonarLintOutputLogger(serviceProvider), new FileWrapper());
```

Comment:
I extracted the existing code to read from a config file to a helper class. To make it more testable, I changed the extract code to use the _IFile_ abstraction, which meant I had to make this class use the _FileWrapper_ too.
However, I haven't changed the existing _ISolutionBindingSerializer_ tests that actually read and write files from disk - low ROI.